media-video/mkvtoolnix/mkvtoolnix-73.0.0.recipe

```diff
@@ -0,0 +1,129 @@
+SUMMARY="A Matroska file creator and editor"
+DESCRIPTION="MKVToolNix is a set of tools to create, alter and inspect \
+Matroska files. Matroska is a versatile media format which can contain \
+video, audio and subtitles, as well as chapter lists."
+HOMEPAGE="https://mkvtoolnix.download"
+COPYRIGHT="2003-2023 Moritz Bunkus"
+LICENSE="GNU GPL v2"
+REVISION="2"
+SOURCE_URI="https://mkvtoolnix.download/sources/mkvtoolnix-$portVersion.tar.xz"
+SOURCE_DIR="mkvtoolnix-$portVersion"
+CHECKSUM_SHA256="f31a129723571b46a974bc5d57d73733c1245ee429afd6ddaf274038e94e2280"
+PATCHES="mkvtoolnix-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+ADDITIONAL_FILES="
+	mkvtoolnix-gui.rdef.in
+"
+
+PROVIDES="
+	mkvtoolnix$secondaryArchSuffix = $portVersion
+	app:MKVToolNixGUI$secondaryArchSuffix = $portVersion
+	cmd:mkvtoolnix_gui$secondaryArchSuffix = $portVersion
+	cmd:mkvmerge$secondaryArchSuffix = $portVersion
+	cmd:mkvextract$secondaryArchSuffix = $portVersion
+	cmd:mkvinfo$secondaryArchSuffix = $portVersion
+	cmd:mkvpropedit$secondaryArchSuffix = $portVersion
+"
+
```

Comment:
No empty line between PROVIDES and REQUIRES, same for BUILD_REQUIRES and BUILD_PREREQUIRES